### PR TITLE
Order models alphabetically in defineWorkspace

### PR DESCRIPTION
**Pull request by adrianclay**
## What does this pull request do?

Order models alphabetically in defineWorkspace

## What is the intent behind these changes?

Refactoring

### CODE REVIEW

**Comment by richrace** [comment]
I'd want to see how this looks on structurizr see: 

https://github.com/ministryofjustice/laa-architecture-as-code/pull/51

**Comment by adrianclay** [comment]
Oh dear oh dear.  I don't think ordering should impact this.  How is that desirable behaviour? 🤦

**Comment by heidar** [comment]
hmm that's annoying

**Comment by sldblog** [comment]
To be honest, this might be undefined behaviour. It's worth popping an issue on https://github.com/structurizr/java to clarify the intent.

File: src/main/kotlin/defineWorkspace.kt [hunk 1]
```diff
@@ -8,43 +8,43 @@ import com.structurizr.model.Model
...
+  CCMS,
+  CCR,
+  CDA,
+  CFE,
   CIS,
+  CLA,
   CWA,
-  SendGrid,
-  TAD,
+  CommonPlatform,
```
Comment (oliverweighell) [inline src/main/kotlin/defineWorkspace.kt:25]: Might as well alphabetise beyond first letter? If so, Common and Corporate above CWA.

File: src/main/kotlin/defineWorkspace.kt [hunk 2]
```diff
@@ -8,43 +8,43 @@ import com.structurizr.model.Model
...
   CorporateDocumentServices,
-  MLRA,
-  Libra,
-  InfoX,
-  ERIC,
   EDW,
-  Rossendales,
-  Allpay,
+  ERIC,
   Eckoh,
```
Comment (oliverweighell) [inline src/main/kotlin/defineWorkspace.kt:29]: Eckoh above ERIC

File: src/main/kotlin/defineWorkspace.kt [hunk 3]
```diff
@@ -8,43 +8,43 @@ import com.structurizr.model.Model
...
-  Barclaycard
+  EligibilityCalculator,
+  FALA,
+  GOVUKNotify,
+  Geckoboard,
```
Comment (oliverweighell) [inline src/main/kotlin/defineWorkspace.kt:33]: Gecko above GOV

File: src/main/kotlin/defineWorkspace.kt (hunk 1 above)
Comment (oliverweighell) [reply src/main/kotlin/defineWorkspace.kt:25]: Ah, just seen the comments above!

**Comment by richrace** [comment]
I've verified the ordering in the models generates different views.

This is from the PR:
structurizr-61040-cda-container (2)


And this is with `CommonPlatform` model above the `CCCD` model:
structurizr-61040-cda-container (3)

Note: If `CDA` is after `CCCD` is doesn't render "nicely"

**Comment by adrianclay** [comment]
How did you generate those views?  When I try by using the `script/generate_images.sh` it generates this image with either output:

image

**Comment by adrianclay** [comment]
Closing for now given low priority and unwanted side effects.

**Pull request closed without merging**